### PR TITLE
Enable JUnit4 tests except those which require MySQL

### PR DESCRIPTION
**Pull request by ddugovic**
For whatever reason, neither IDEA nor GitHub invoke existing or new JUnit5 tests.  I think this may be due to a Maven versioning issue, so I can leave this PR open until JetBrans and Microsoft offer a compatible Maven version?
https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven

### CODE REVIEW

**Comment by ddugovic** [comment]
Okay, apparently JUnit4 is easier to use, and apparently the GitHub-default MySQL (or other versions) do not work well in GitHub Actions (for example, attempting to create the database) so it is necessary to disable the database authentication test.

However, other JUnit4 tests can be seen working in the pipeline:
https://github.com/Tellmarch/playshogi/actions/runs/196914025

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -16,6 +16,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
+    - name: Install JDBC driver
```
Comment (Tellmarch) [inline .github/workflows/maven.yml:20]: Let's not do that since we skip SQL test

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,381 @@
+-- MySQL Script generated by MySQL Workbench
```
Comment (Tellmarch) [inline schema.sql:1]: I guess this file is not needed for this pull request - if we were to add such files, it would rather be under the sql directory

Comment (ddugovic) [reply schema.sql:1]: Thanks, I will remove this file as it is no longer required.  But yes, now I see the "mysql" directory.

**Pull request merged**